fileparsers/include/DeviceInfoMap.h

```diff
@@ -95,18 +95,17 @@ namespace ChimeraTK {
          * number of reported problems only to critical errors or wants to report
          * all detected problems (errors and warnings)
          */
-        typedef enum {
+        using TYPE = enum {
```

Comment:
Isn't this completely obsolete with C++ and just to avoid the C-ism of explicitly having to do "enum TYPE" in declarations?